Week1/Homework/round.js

```diff
@@ -0,0 +1,6 @@
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let maxNumber = Math.max(a, z);
+console.log(maxNumber);
```

Comment:
Ah. We can use `const` here instead of `let` safely since no re-assignment is being done.